### DISCUSSION TITLE
Do not load chunks outside worldborder

### DISCUSSION THREAD

**Discussion by Emibergo02** [PurpurMC/Purpur#1301] [__Dismissed__] [answered]
I made a world with precise worldborder coordinates to make sure that the world has all the region files inside it.
Is it possible to add a feature to not load chunks/regions outside the border?
i've seen #270 but it is not the same question
Since i do not know where to start, i am asking you for help.
Thank you very much

**Comment by granny** [comment, marked as answer]
I'm a bit lost with how your question is not the same one as in #270. How is "Do not load chunks outside worldborder" different from "Don't load/generate chunks outside of the worldborder"? pop4959's answer still applies.
